CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing
+
+Hello and thank you for your interest in contributing to the 1Password `connect-helm-chart` project 👋! We sincerely appreciate the time and effort you put into improving our Helm Charts. Before you start, please take a moment to read through this guide to understand our contribution process.
+
+## Getting Started
+
+To start contributing, you'll need to fork this repository and clone it locally. Next,
```

Comment:
Ton contirubute, folks won't actually need to fork the repo, they can just work directly from it :)